packages/cosmos/build

```diff
@@ -9,6 +9,8 @@ chmod -R o+r "${PKG_PATH}/usr"
 cosmos_service="${PKG_PATH}/dcos.target.wants_master/dcos-cosmos.service"
 mkdir -p $(dirname "$cosmos_service")
 
+source /opt/mesosphere/etc/cosmos.env
```

Comment:
I am open to suggestions around this. Do I source the file in the actual unit and replace `$COSMOS...` with `\$COSMOS...`?